### PR TITLE
It2 league stats

### PR DESCRIPTION
**Pull request by abcdefghijohn**
First test written along with method. Used actual csv file since team.csv only has 33 lines of data.

### CODE REVIEW

File: lib/team.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 class Team
```
Comment (TravisGM92) [inline lib/team.rb:1]: I like how you created a new class 'Team'. Perhaps you can help me do that with my code so we can cut down on how long it is. We'll all need to talk once we wanna start merging

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -1,14 +1,9 @@
...
-  attr_reader :data
-  def self.from_csv(data)
-    StatTracker.new(data)
+  def self.from_csv(object, path)
+    CSV.foreach(path, headers: true, header_converters: :symbol) do |row|
+      object.add(object.new(row))
+    end
   end
-
-  def initialize(data)
```
Comment (TravisGM92) [inline lib/stat_tracker.rb:10]: I see you got rid of initialize. Mind explaining why? Not that I don't approve, just wanna know why you did. I've never had to do that

Comment (abcdefghijohn) [reply lib/stat_tracker.rb:10]: I thought if I had to instantiate  everything within the stat tracker I'd end up with too much so I just created a separate team class and plan to create the additional necessary classes for future methods as well.

**Comment by TravisGM92** [comment]
I think we should merge John's branch first. Then, when that still works well, we can merge the next branch together. What do you all think?

**Comment by TravisGM92** [comment]
We will be reorganizing our code so that there is more uniformity. It would be better if we all had similar looking ways to access our CSV files.

**Pull request closed without merging**